data/T2D.csv

```diff
@@ -1,404 +1,459 @@
+Type 2 Diabetes Mellitus	9	81694033	G	A	0.067658648473815		4.40E-24	10.1038/s41588-018-0241-6	rs2796441		
+Type 2 Diabetes Mellitus	9	94239400	A	C	0.039220713153281		2.20E-09	10.1038/s41588-018-0241-6	rs55653563		
+Type 2 Diabetes Mellitus	9	94735212	T	C	0.139761942375159		3.40E-06	10.1038/s41588-018-0241-6	rs12236906		
+Type 2 Diabetes Mellitus	9	133273813	C	T	0.048790164169432		3.90E-12	10.1038/s41588-018-0241-6	rs505922		
+Type 2 Diabetes Mellitus	9	136341149	G	C	0.058268908123976		1.20E-06	10.1038/s41588-018-0241-6	rs78403475		
+Type 2 Diabetes Mellitus	9	136346577	G	A	0.086177696241053		6.70E-26	10.1038/s41588-018-0241-6	rs28505901		
+Type 2 Diabetes Mellitus	9	136612760	C	T	0.039220713153281		2.20E-07	10.1038/s41588-018-0241-6	rs11793035		
+Type 2 Diabetes Mellitus	9	136842636	A	G	1.007957920399980		7.90E-06	10.1038/s41588-018-0241-6	9:139737088:G:A		
+Type 2 Diabetes Mellitus	10	12265895	T	C	0.086177696241053		1.50E-32	10.1038/s41588-018-0241-6	rs11257655		
+Type 2 Diabetes Mellitus	10	69561523	G	A	0.067658648473815		6.60E-18	10.1038/s41588-018-0241-6	rs177045		
+Type 2 Diabetes Mellitus	10	69561902	C	G	0.039220713153281		7.30E-06	10.1038/s41588-018-0241-6	rs61850200		
+Type 2 Diabetes Mellitus	10	69572545	T	C	0.086177696241053		1.50E-06	10.1038/s41588-018-0241-6	rs41277236		
+Type 2 Diabetes Mellitus	10	69587555	T	C	0.444685821261446		4.70E-07	10.1038/s41588-018-0241-6	rs549498088		
+Type 2 Diabetes Mellitus	10	69706822	G	T	0.048790164169432		2.20E-14	10.1038/s41588-018-0241-6	rs2642588		
+Type 2 Diabetes Mellitus	10	79193069	G	C	0.067658648473815		1.70E-29	10.1038/s41588-018-0241-6	rs703972		
+Type 2 Diabetes Mellitus	10	79336832	G	A	0.039220713153281		1.80E-06	10.1038/s41588-018-0241-6	rs1317617		
+Type 2 Diabetes Mellitus	10	88009583	T	C	0.058268908123976		4.70E-08	10.1038/s41588-018-0241-6	rs11202627		
+Type 2 Diabetes Mellitus	10	92164906	T	C	0.029558802241545		4.10E-07	10.1038/s41588-018-0241-6	rs7078559		
+Type 2 Diabetes Mellitus	10	92702670	T	C	0.058268908123976		1.40E-08	10.1038/s41588-018-0241-6	rs10882101		
+Type 2 Diabetes Mellitus	10	92719350	A	G	0.058268908123976		5.00E-07	10.1038/s41588-018-0241-6	rs1112718		
+Type 2 Diabetes Mellitus	10	112940076	G	C	0.405465108108164		2.60E-08	10.1038/s41588-018-0241-6	rs536643418		
+Type 2 Diabetes Mellitus	10	112943203	A	G	0.307484699747961		2.20E-08	10.1038/s41588-018-0241-6	rs140242150		
+Type 2 Diabetes Mellitus	10	112943377	T	C	0.058268908123976		2.00E-15	10.1038/s41588-018-0241-6	rs7918400		
+Type 2 Diabetes Mellitus	10	112980578	C	G	0.190620359608650		1.20E-13	10.1038/s41588-018-0241-6	rs184509201		
+Type 2 Diabetes Mellitus	10	112991414	G	A	0.157003748809665		6.10E-06	10.1038/s41588-018-0241-6	rs180988137		
+Type 2 Diabetes Mellitus	10	112998197	C	G	0.048790164169432		1.60E-07	10.1038/s41588-018-0241-6	rs78025551		
+Type 2 Diabetes Mellitus	10	112998590	T	C	0.314810739840034		0	10.1038/s41588-018-0241-6	rs7903146		

```

Comment:
The actual p-value is 5.8x10-447. Maybe capping it at the limit, 1e-300 would be prudent.

Reply:
I added p = 1e-300 and note that actual p-value is 5.8x10-447.